spec/integration/integration_spec.rb

```diff
@@ -901,5 +901,28 @@ class Tag
         end
       end
     end
+
+    describe 'overriden changes_method with non_database_fields' do
```

Comment:
That's not really important, though. The point of the change was to be able to provide a custom change array on a field that wasn't necessarily in the database. Mongoid doesn't track changes on an ``attr_accessor`` and so a test on that would not be testing the gem code, but only testing what you wrote in the test itself.

Does that explanation make any sense?